app/src/main/java/org/mozilla/fenix/home/SearchView.kt

```diff
@@ -8,15 +8,16 @@ import android.content.Context
 import android.graphics.drawable.TransitionDrawable
 import android.util.AttributeSet
 import android.widget.FrameLayout
+import androidx.core.content.res.ResourcesCompat
 import org.mozilla.fenix.R
 
 class SearchView(context: Context, attrs: AttributeSet) : FrameLayout(context, attrs) {
     private val lightDrawable =
-        resources.getDrawable(R.drawable.home_search_background_normal, context.theme)
+        ResourcesCompat.getDrawable(resources, R.drawable.home_search_background_normal, context.theme)
```

Comment:
Like `ContextCompat`, `ResourcesCompat` doesn't work with vector compat. (Why are there so many ways to get a drawable? 🤷‍♀) Use `AppCompatResources.getDrawable` with the context property on the view.